### ISSUE TITLE
Having a room in GM 8.1 with the "Clear Background with Window Color" option unchecked causes loading to fail

### ISSUE THREAD

**Issue by fcard** [IsmAvatar/LateralGM#443]
Even in a newly created project, having the following option unchecked
![The "Clear Background With Window Color" prominently surrounded by a red rectangle, in the "Views" tab in the room settings](https://user-images.githubusercontent.com/12222927/59561171-5f49ae80-8ff3-11e9-93e2-5841b768e23b.png)
I get the error
```
LateralGM Version: 1.8.82
Working Directory: /home/phicr/Downloads/lateralgm.jar

Operating System: Linux
Version: 4.19.0-5-amd64
Architecture: amd64

Java Name: OpenJDK 64-Bit Server VM
Java Vendor: Oracle Corporation
Version: 1.8.0_212

Available processors (cores): 4
Free memory (bytes): 88561016
Maximum memory (bytes): 1834483712
Total memory available to JVM (bytes): 132644864

File system root: /
Total space (bytes): 981283790848
Free space (bytes): 801128706048
Usable space (bytes): 751210827776

Stack trace:
org.lateralgm.file.GmFormatException: java.io.IOException: invalid boolean data: 3 at filepos: 144[65]
	at org.lateralgm.file.GmStreamDecoder.readBool(GmStreamDecoder.java:149)
	at org.lateralgm.file.GmFileReader.readRooms(GmFileReader.java:1030)
	at org.lateralgm.file.GmFileReader.readProjectFile(GmFileReader.java:260)
	at org.lateralgm.file.GmFileReader.readProjectFile(GmFileReader.java:168)
	at org.lateralgm.main.FileChooser$ProjectReader.read(FileChooser.java:391)
	at org.lateralgm.main.FileChooser$1.run(FileChooser.java:630)
	at java.lang.Thread.run(Thread.java:748)
```
After that I end up with unordered, ungrouped resources, and only the rooms read before the offending one complete. The offending room itself is present but incomplete (no tiles/objects, and background settings are incomplete, maybe some others)

**Comment by RobertBColton** [comment]
Excellent work, I confirm I can reproduce, thanks for this one.
1) Open new game in GM8.1
2) Add room and uncheck Views->"Clear Background with Window Color"
3) Save room and game as gm81
4) Open in LateralGM

**Comment by RobertBColton** [comment]
Not a regression, I can reproduce all the way back to lgm16b4. I'll try to get this one fixed.

**Comment by RobertBColton** [comment]
Closing as resolved by #446. You can now obtain a release of LateralGM version 1.8.84 with the fix.
